### ISSUE TITLE
 can not join because of failed to synchronize registry data from server message (server with friends)

### ISSUE THREAD

**Issue by Krimm-Sala-Bimm** [Iskallia/Vault-public#582]
hi i have bought a server to play vh with friends but i ran into some problems while setting up 
the server i downloaded vh on curseforge because the server i bought sayed you could install mods from curseforge so i did 
download to server went good and is running but if i join the server i get this message (failed to synchronize registry data from server) 
i searched on the internet for a fix but they say i have to install every mod one by one its like 117 mods but then i only have the mods not the config filles 
what means its not realy vh how it supposed to be is there a other solution for tis specifically problem or has someone had tis problem to

**Comment by joshuaepstein** [comment]
Have you installed the server pack from the curseforge page

**Comment by NickTehCoder** [comment]
yeah, i have the same issue and i installed the server pack from curseforge

**Comment by ShoiLyaung** [comment]
same problem
